cilium/CFP-20550-cidr-policy-for-nodes.md

```diff
@@ -0,0 +1,80 @@
+# CFP-20550: CIDR policy for nodes
+
+**SIG: SIG-Policy**
+
+**Begin Design Discussion:** 2023-04-17
+
+**Cilium Release:** 1.14 or 1.15
+
+**Authors:** Joe Stringer <joe@cilium.io>
```

Comment:
```suggestion
**Authors:** Joe Stringer <joe@cilium.io>

**Status:** Dormant
```

Reply:
:+1: IIRC @squeed ended up implementing this feature using this CFP as input, but the implementation may have diverged. Simplest low effort step is just close this PR. Ideally we would update it to reflect the implemented functionality and move it to `status: released`.